lib/internal/fs/cp/cp-sync.js

```diff
@@ -0,0 +1,349 @@
+'use strict';
+
+// This file is a modified version of the fs-extra's copySync method
+// modified in the following ways:
+//
+// - Use of the graceful-fs has been replaced with fs.
+// - Helpers from stat.js have been inlined.
+// - Formatting and style changes to match Node.js' linting rules.
+// - Parameter validation has been added.
```

Comment:
I think the modifications are more substantial now? Maybe we should remove the list?

Reply:
I've removed, but left the attribution to `fs-extra` for posterity.